voucher/urls.py

```diff
@@ -0,0 +1,12 @@
+"""
+Voucher URL patterns
+"""
+from django.urls import path
+from voucher import views
+
+urlpatterns = [
+    path("/upload", views.UploadVoucherFormView.as_view(), name="upload"),
+    path("/resubmit", views.resubmit, name="resubmit"),
+    path("/redeemed", views.redeemed, name="redeemed"),
+    path("/enroll", views.EnrollView.as_view(), name="enroll"),
```

Comment:
```
path("upload/"....
path("resubmit/"...
```
etc.

That way, entering `http://xpro.odl.local:8053/voucher/upload` will redirect to `http://xpro.odl.local:8053/voucher/upload/` and entering `http://xpro.odl.local:8053/voucher/upload/` will no longer return a 404.

Reply:
Oh that's what you meant by the slash. My mistake